extensions/mssql/src/objectManagement/ui/databaseDialog.ts

```diff
@@ -869,7 +875,8 @@ export class DatabaseDialog extends ObjectManagementDialogBase<Database, Databas
 		this.filestreamFilegroupNameContainer = await this.getFilegroupNameGroup(this.filestreamFilegroupsTable, FileGroupType.FileStreamDataFileGroup);
 		const addButtonComponent: DialogButton = {
 			buttonAriaLabel: localizedConstants.AddFilegroupText,
-			buttonHandler: () => this.onAddDatabaseFileGroupsButtonClicked(this.filestreamFilegroupsTable)
+			buttonHandler: () => this.onAddDatabaseFileGroupsButtonClicked(this.filestreamFilegroupsTable),
+			enabled: this.viewInfo.serverFilestreamAccessLevel !== null && this.viewInfo.serverFilestreamAccessLevel !== 0
```

Comment:
Would suggest making a helper function for this check since you use it in multiple places